tests/data/reframe

```diff
@@ -1,75 +1,71 @@
 data = [
     {
-        "raw": "<12>1 2021-01-18T04:59:09.645983+01:00 node113 reframe: - reframe:  username=vsc10009|version=3.3|commit=0895565|name=OSUBandwidthTest|system=hydra|partition=skylake-mpi|environ=foss-2020a|num_tasks=2|num_tasks_per_node=1|tags=prod_small|jobid=3721799.master01.hydra.brussel.vsc|perf_var=bandwidth|perf_value=9485.01|ref=11500|lower=-0.1|upper=1.5|unit=MB/s",
+        "raw": "2021-12-23T15:20:18.050421+01:00 login1 reframe: username=vsc10009|version=3.9.1|commit=|name=OSUBandwidthTest|system=hydra|partition=skylake-mn-mpi-ib|environ=foss-2021a|num_tasks=2|num_cpus_per_task=1|num_tasks_per_node=1|modules=|jobid=6090034|perf_var=bandwidth|perf_value=3667.21|unit=MB/s",
         "expected": {
-            "environ": "foss-2020a",
-            "jobid": "3721799.master01.hydra.brussel.vsc",
-            "lower": -0.1,
+            "environ": "foss-2021a",
+            "jobid": "6090034",
             "name": "OSUBandwidthTest",
             "num_tasks": 2,
+            "num_cpus_per_task": 1,
             "num_tasks_per_node": 1,
-            "partition": "skylake-mpi",
-            "perf_value": 9485.01,
+            "partition": "skylake-mn-mpi-ib",
+            "perf_value": 3667.21,
             "perf_var": "bandwidth",
             "program": "reframe",
-            "ref": 11500.0,
-            "@source_host": "node113",
+            "@source_host": "login1",
             "system": "hydra",
-            "tags": "prod_small",
-            "@timestamp": "2021-01-18T03:59:09.645Z",
+            "modules": "",
```

Comment:
it fails on this. Modules is empty so it doesn't get set. Do we want this?

Reply:
I've made modules optional, so I can just remove this line, right?

Reply:
yes, some tests don't load any additional modules to run